packages/react-scripts/template/__mocks__/store.js

```diff
@@ -0,0 +1,7 @@
+export const locales = {
```

Comment:
widziałem jeszcze gdzieś w jakimś projekcie, że zamiast pakować to do `__mocks__` to był katalog `fixtures` i tam cała implementacja tego pod testy. Może warto to też wprowadzić?

Reply:
w glastonbury - jestem za

Reply:
czyli w tym przypadki macie na myśli żeby po prostu zmienić nazwe folderu na `fixtures`? Czy coś więcej?

Reply:
Zmieniam nazwe folderu na `fixtures`, jesli chcemy dodac coś więcej, możemy osobną PRkęz tym zrobić